script/scaleUpMesh/scaleUpMesh.py

```diff
@@ -0,0 +1,100 @@
+import regex
+
+
+class Node:
+    idx: int
+    x: float
+    y: float
+    z: float
+
+    def __init__(self, idx: int, x: float, y: float, z: float):
+        self.idx = idx
+        self.x = x
+        self.y = y
+        self.z = z
+
+    @staticmethod
+    def from_line(line):
+        results = regex.search(r"^(\d+)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)\s([+-]?[0-9]*\.[0-9]+(?:e-?\d+)?)", line)
```

Comment:
Wow, very sophisticated with `regex`.

Reply:
Not necessary 😄. This wasn't meant as a negative comment by any terms. However, such solutions can usually be quite error-prone when it's literally just one float after another separated by white-characters.